h2o-bindings/bin/custom/python/gen_targetencoder.py

```diff
@@ -15,8 +15,22 @@ def transform(self, frame, data_leakage_handling="None", noise=-1, seed=-1):
         :param int seed: a random seed used to generate draws from the uniform distribution for random noise. Defaults to -1.
 
         :example:
-        >>> targetEncoder = TargetEncoder(encoded_columns=te_columns, target_column=responseColumnName, blended_avg=True, inflection_point=10, smoothing=20)
-                            >>> encodedTrain = targetEncoder.transform(frame=trainFrame, data_leakage_handling="None", seed=1234, is_train_or_valid=True)
+        >>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+        >>> predictors = ["home.dest", "cabin", "embarked"]
+        >>> response = "survived"
+        >>> titanic["survived"] = titanic["survived"].asfactor()
```

Comment:
I guess users using TE should know this by now – that response needs to be converted to a categorical to enforce classification –, but let's make it even more explicit in the examples and reuse the variable just defined:
`titanic[response] = titanic[response].asfactor()`